draft-ietf-jsonpath-base.md

```diff
@@ -1545,7 +1565,7 @@ $[?length(@.authors) >= 5]
 ~~~
 
 Its only argument is an instance of `ValueType` (possibly taken from a
-singular query as in the example above).  The result also is an
+singular query, as in the example above).  The result also is an
```

Comment:
A similar parenthetical remark should be added to the descriptions of the "match" and "search" function extensions. Cf. PR #437.

Reply:
Done.